ui/src/config.js

```diff
@@ -47,6 +51,10 @@ const exports = {
     SLATE_SCRATCH: "/N/scratch",
     SLATE_SCRATCH_FILES: "/N/scratch files",
   },
+  auth_enabled: {
```

Comment:
Is it possible to change this to a more generic key like `features_enabled` instead of `auth_enabled`, so we can reuse this block for enabling/disabling future features?

Reply:
Yes, this can be folded into the feature flags once that is merged.